AirportTicketBooking/InputModels/InputUser/InputUser.cs

```
@@ -5,9 +5,35 @@ namespace AirportTicketBooking.InputModels.InputUser;
 
 public static class InputUser
 {
-    public static User InputManager()
+    public static User Input()
     {
-        var manager = new Manager {
+        Console.WriteLine("Enter 1 if you want to create Manager, and 2 if you want to create Passenger");
+        var input = Console.ReadLine();
+        int number;
+        while (string.IsNullOrWhiteSpace(input) || !int.TryParse(input, out number) || (number != 1 && number != 2))
+        {
+            Console.WriteLine("invalid input, try again");
+            input = Console.ReadLine();
+        }
+
+        const int managerNumber = 1, passengerNumber = 2;
+        switch (number)
+        {
+            case managerNumber:
+                return InputManager();
+                break;
+            case passengerNumber:
+                return InputPassenger();
+                break;
+            default:
+                return null;
```

instead of null, please throw an exception.